utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -27,8 +27,9 @@ case class KafkaConfig(
     schemaRegistryCacheConfig: SchemaRegistryCacheConfig = SchemaRegistryCacheConfig(),
     avroAsJsonSerialization: Option[Boolean] = None,
     kafkaAddress: Option[String] = None,
-    // TODO: remove this feature flag in future release
-    useNamingStrategyForConsumerGroupId: Boolean = true
+    // TODO question: move it into a generic map / move it into nested config class / leave it here?
```

Comment:
Not sure about this - its a similar concept to maxOutOfOrderness which is configured in `kafkaEspProperties` - but I don't see a reason besides consistency to have this in a generic string map